snap/snapcraft.yaml

```diff
@@ -1,42 +1,44 @@
 name: shotwell
-version: '0.32.1'
```

Comment:
This is necessary for the CI to run as it is currently written as you can see with the smoke failure